client/signup/config/flows.js

```diff
@@ -164,11 +164,16 @@ function getCurrentFlowNameFromTest( currentURL ) {
 		return ( 'noSurvey' === abtest( 'verticalSurvey' ) ) ? 'main' : 'verticals';
 	}
 
-	// Only consider users from the general /start path.
+	// Only consider users from the WordPress.com homepage for the DSS test.
 	if ( '/start/en?ref=homepage' === currentURL && 'dss' === abtest( 'dss' ) ) {
 		return 'dss';
 	}
 
+	// Consider remaining users for the Triforce AB test.
+	if ( 'notTested' === abtest( 'dss' ) && 'triforce' === abtest( 'triforce' ) ) {
```

Comment:
I'm a little fuzzy on some of the details, but won't this call to `abtest( 'dss' )` assign all users to a variation of the `dss` test even if they didn't come from the home page?